src/components/forms/FormRow/index.tsx

```diff
@@ -34,13 +34,14 @@ const FormRowBase = (props: FormRowProps): JSX.Element => {
       id: formId,
       ...rest
     };
-    if (type == 'text') return <FormTextInput {...component_props} />;
-    else if (type == 'textarea') return <FormTextArea {...component_props} />;
-    else if (type == 'async-select') return <FormAsyncSelect {...component_props} />;
-    else if (type == 'select') return <FormSelect {...component_props} />;
-    else if (type == 'createable-select') return <FormCreateableSelect {...component_props} />;
-    else if (type == 'dropzone') return <FormDropzone {...component_props} />;
-    else if (type == 'checkbox') return <FormCheckbox {...component_props} />;
+    if (type === 'text') return <FormTextInput {...component_props} />;
+    else if (type === 'password') return <FormTextInput {...component_props} />;
+    else if (type === 'textarea') return <FormTextArea {...component_props} />;
+    else if (type === 'async-select') return <FormAsyncSelect {...component_props} />;
+    else if (type === 'select') return <FormSelect {...component_props} />;
+    else if (type === 'createable-select') return <FormCreateableSelect {...component_props} />;
+    else if (type === 'dropzone') return <FormDropzone {...component_props} />;
+    else if (type === 'checkbox') return <FormCheckbox {...component_props} />;
```

Comment:
This series of nested ifs doesn't look good... How about using `switch` statement instead? 
Alternatively, you can use a simple map: 
```
const inputsMap: Record<FormType, React.Node> = {
  text: <FormTextInput {...component_props} />,
  password: <FormTextInput {...component_props} />,
  /* ... */
}
if (inputsMap[type]) return inputsMap[type];
```
BTW why `component_props` is in snake case?

Reply:
Not my component, just using it

Reply:
That doesn't mean it cannot be improved